openlibrary/plugins/upstream/account.py

```diff
@@ -225,20 +227,27 @@ def is_plugin_enabled(self, name):
     def POST(self):
         i = web.input('email', 'password', 'username', agreement="no")
         i.displayname = i.get('displayname') or i.username
-
+        
         f = self.get_form()
+        
```

Comment:
Suggestion: we might declare a variable `page = None` before checking e.g. validation etc and then at the bottom (before returning) set `page.v2 = True` once? (DRY: don't repeat ourselves)

Reply:
The CI problem was an issue on master which has since been fixed.

I've restarted the build. You may need to rebase it against upstream.